### ISSUE TITLE
World layer in Querty

### ISSUE THREAD

**Issue by frulio** [sunaku/glove80-keymaps#15]
Hi, first I just want to thank you for the **amazing** work, this configuration have been working great for me.

**The backgroud**: I work in devops and speak spanish, the code and commands are all written in english, but the normal communication with my team mates is in spanish.

**The problem**:  by default I use querty as my layer for typing to reduce the friction when I need to use another keyboard and is the one I'v used for years (you know is hard to change and be productive at the same time), when I need to type  "funcionó" I have to press shif+quotes then space (if using the dead keys configuration) for the ó quote+o, but there is a world layer and I just can press modifier + e, which in the default configuration makes sense because the o char is there, but in querty is not.

is there a way to make it match the character and not the position? so it works in every layout. Also this will be helpful because I can program and use the quotes normally (or just press space + e which I love btw). 

Having that will allow me to disable deakeys (I don't use spanish layout keyboards) and use the quotes directly but have access to special characters

I'm new with the keyboard, so no much experience here.

**Comment by sunaku** [comment]
I don't think this is possible because letter placement can be wildly different across alpha layouts and this would make the design of the World layer very complicated: currently, it has modifiers on the right hand and characters on the left hand.  For example, consider the scenario of your request: if O (on the right hand in QWERTY) should hold the World O character cluster, where should RALT be moved instead?  So for this reason, I think this can't be implemented in a generic way and you may be better served by adjusting the World layer (or creating another custom layer) to suit your needs.  Sorry for the inconvenience.
